src/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { ClientAuthMethod, ResponseType } from 'openid-client'
+
+export const SGID_SIGNING_ALG = 'RS256'
```

Comment:
Suggestion: Should we have block comments to group our constants? What I had in mind was something similar to how we group constants in `sgid-server` - Could be just a phrase, eg. 'PKCE', 'API versioning' etc.

Reply:
yup okay that makes sense

Reply:
Refactored in this [PR](https://github.com/opengovsg/sgid-client/pull/39) in order to prevent this PR from growing too large